application/src/main/java/org/opentripplanner/graph_builder/GraphBuilderDataSources.java

```diff
@@ -103,20 +106,33 @@ public boolean has(FileType type) {
     return inputData.containsKey(type);
   }
 
-  public Iterable<DataSource> get(FileType type) {
-    return inputData.get(type);
+  public Iterable<ConfiguredDataSource<OsmExtractParameters>> getOsmConfiguredDatasource() {
+    return ofStream(OSM).map(this::mapOsmData).toList();
   }
 
-  public Iterable<ConfiguredDataSource<OsmExtractParameters>> getOsmConfiguredDatasource() {
-    return inputData
-      .get(OSM)
-      .stream()
-      .map(it -> new ConfiguredDataSource<>(it, getOsmExtractConfig(it)))
-      .toList();
+  public Iterable<ConfiguredDataSource<DemExtractParameters>> getDemConfiguredDatasource() {
+    return ofStream(DEM).map(this::mapDemData).toList();
   }
 
-  private OsmExtractParameters getOsmExtractConfig(DataSource dataSource) {
-    return buildConfig.osm.parameters
+  public Iterable<ConfiguredCompositeDataSource<GtfsFeedParameters>> getGtfsConfiguredDatasource() {
+    return ofStream(GTFS).map(this::mapGtfsFeed).toList();
+  }
+
+  public Iterable<
+    ConfiguredCompositeDataSource<NetexFeedParameters>
+  > getNetexConfiguredDatasource() {
+    return ofStream(NETEX).map(this::mapNetexFeed).toList();
+  }
+
+  /**
+   * Returns the optional data source for the stop consolidation configuration.
+   */
+  public Optional<DataSource> stopConsolidation() {
+    return store.stopConsolidation();
+  }
+
+  private ConfiguredDataSource<OsmExtractParameters> mapOsmData(DataSource dataSource) {
```

Comment:
This method should be lower in the file as it's private. The diff is already difficult to read so it shouldn't matter much if you move one more method a bit.